pkg/cmd/codespace/list.go

```diff
@@ -43,6 +43,13 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 		return fmt.Errorf("error getting codespaces: %w", err)
 	}
 
+	hasNonProdVSCSTarget := false
+	for _, apiCodespace := range codespaces {
+		if apiCodespace.VSCSTarget != "prod" {
+			hasNonProdVSCSTarget = true
```

Comment:
Should this break out after finding the first non-production one?

Reply:
Yup, added